format/nexus.py

```diff
@@ -81,6 +83,45 @@ class NXValidationError(RuntimeError):
     """A specific exception to record validation errors"""
 
 
+def find_classes(node: NXNode, *nx_classes: Optional[str]) -> Tuple[List[h5py.Group]]:
+    """
+    Find instances of multiple NXclass types within the children of the current node.
+
+    Args:
+        node: The input h5py node (h5py.File or h5py.Group).
+        nx_classes: Names of NXclass types to search for.
+
+    Returns:
+        A list of matching nodes for each of the specified NXclass types.
```

Comment:
In any case, this needs correcting.
```suggestion
        A list of matching nodes for each of the specified NX_class types.
```